### DISCUSSION TITLE
The new documentation was a mistake and should be reverted

### DISCUSSION THREAD

**Discussion by aljawaid** [kanboard/kanboard#5127] [General]
It makes it harder to read with such a dark background.  Dark backgrounds may be suitable for writing code, but not for instructions/manuals.  It looks too heavy.

Information seems to be missing or less clear than previous documentation versions/styles.

@fguillot Please consider, a few users in the forum have also disliked the sudden documentation change.  It was already lacking content updates/accuracy and this new design has made it harder to refer to.

**Comment by aljawaid** [comment]
https://kanboard.discourse.group/t/what-a-about-website-and-theme-refresh-and-improve/2547/4?u=aljawaid

**Reply by aljawaid** [reply]
https://kanboard.discourse.group/t/what-a-about-website-and-theme-refresh-and-improve/2547/3?u=aljawaid

**Reply by aljawaid** [reply]
that's two different opinions other than mine

**Comment by nekohayo** [comment]
What are you referring to, exactly? Looking at https://docs.kanboard.org, as it is daytime for me right now, it's all black-on-white (light color scheme) for me...  maybe you've been looking at it in the nighttime with your operating system set to dark mode and your browser is set to automatically inherit it, in which case I can see that https://docs.kanboard.org/stylesheet.css's `@media (prefers-color-scheme: dark)` will switch the color scheme to dark mode.

To me that's a feature (and I would want that [for kanboard itself](https://kanboard.discourse.group/t/suggestion-for-the-main-theme-3rd-parties-automatic-dark-light-mode-using-the-browsers-prefers-color-scheme-dark-css-mediaqueries/2181), too). I can't read documentation at night if it's black-on-white instead of white-on-black. If you don't like that, tell your browser to not honor dark mode, or don't turn on your operating system's dark mode? Because that's how a _lot_ of websites will behave from now on, not just kanboard's docs. I would strongly advise against removing that prefers-color-scheme:dark feature from the CSS.

**Reply by aljawaid** [reply]
Thanks, you are correct.... the dark theme disabled and the website looks fine.  Having saidthat, for Kanboard,asit is such lacking in frontend designs and a productivity tool, should not have such a setting set.

My second point was the few users who have complained on the missing or lack of information that wasin the old version.

> Because that's how a lot of websites will behave from now on, not just kanboard's docs.
Yes but only where it is appropriate, I don't think Kanboard is appropriate
